### PR TITLE
[MM-30352] Cypress/E2E: Plugin Management

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

- [x] [MM-T2400](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T2400) - Plugins Management

- [x] [MM-T2391](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T2391) - If plugins fail to start, they can be disabled

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-30352

The linked ticket is supposed to be scoped to the demo plugin, but the two cases in this PR are not directly related to the demo plugin so I've put them in the webapp repo.

### CODE REVIEW

File: e2e/cypress/integration/plugins/plugin_install_spec.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * Copy to : ./e2e/cypress/fixtures/com.mattermost.demo-plugin-0.9.0.tar.gz
+ */
+
+describe('Plugins Management', () => {
+    const pluginID = 'com.mattermost.demo-plugin';
+    const pluginFile = 'com.mattermost.demo-plugin-0.9.0.tar.gz';
+
+    before(() => {
+        cy.apiInitSetup();
+        cy.apiAdminLogin();
```
Comment (saturninoabril) [inline e2e/cypress/integration/plugins/plugin_install_spec.js:27]:
```suggestion
```

File: e2e/cypress/integration/plugins/plugin_install_spec.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+        cy.apiRemovePluginById(pluginID, '');
+    });
+
+    after(() => {
+        cy.apiRemovePluginById(pluginID, '');
```
Comment (saturninoabril) [inline e2e/cypress/integration/plugins/plugin_install_spec.js:33]:
```suggestion
        cy.apiRemovePluginById(pluginID);
    });

    after(() => {
        cy.apiRemovePluginById(pluginID);
```

**Review by saturninoabril — APPROVED** [review]
Thanks @mickmister! Tested and passed. Left couple of suggestions but approving now.

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Tests run green
- Step match the test script

LGTM!
2 Corresponding TM4J task correctly have Cypress status of `Update`

**Pull request merged**